src/Services/FHIR/FhirAllergyIntoleranceService.php

```diff
@@ -106,7 +106,7 @@ public function getOne($id)
     public function createAllergyIntoleranceResource($id = '', $data = '', $encode = true)
     {
         $typeCoding = new FHIRCoding();
-        $typeCoding->setSystem("http://terminology.hl7.org/CodeSystem/allergyintolerance-clinical");
+        $typeCoding->setSystem("https://hl7.org/fhir/allergyintolerance.html");
```

Comment:
Hey @stu01509 Nice that you pointed this but u need to know that these URLs are not meant to be visited they are canonical URL which is used for unique identification. We just copy these URLs from fhir Docs for the related ValueSets.
For further lookup, I am adding some links to spec.
https://www.hl7.org/fhir/valueset-allergyintolerance-clinical.html
https://www.hl7.org/fhir/codesystem.html
https://www.hl7.org/fhir/valueset.html

Also, you can ask @sjpadgett for their review 👍 
Thanks 😃

Reply:
hi @yashrajbothra , makes sense and gonna close this PR.